container/compliance/generate-attributions-md/tests/test_extractor.py

```
@@ -0,0 +1,146 @@
+    assert expanded == ["lib/bar", "lib/foo"]
+
+
+def test_workspace_glob_with_no_matches_returns_empty():
+    assert _expand_workspace_member_paths("nonexistent/*", ["lib/foo/Cargo.toml"]) == []
+
+
+# ---------- _git_read / extract_transitive ----------
+
+
+def _run_git(cwd: Path, *args: str) -> None:
+    subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)